src/applications/travel-pay/containers/TravelPayStatusApp.jsx

```diff
@@ -35,9 +35,8 @@ export default function TravelPayStatusApp({ children }) {
   // and validating logged in status
   // const user = useSelector(selectUser);
 
-  const { isLoading, travelClaims, error } = useSelector(
-    state => state.travelPay,
-  );
+  const { travelClaims } = useSelector(state => state.travelPay);
```

Comment:
This is a minor, subjective semantic thing, but you may be able to avoid some of the digging into `travelClaims` on other lines with destructuring
```js
const {
  travelClaims: { data: claimsData, isLoading, error },
} = useSelector(state => state.travelPay);
```

Reply:
Oh, I see. We may be able to use optional chaining to protect against the `null` approach if you like that one, but I think this way is cool too.